presto-pinot/src/main/java/io/prestosql/pinot/PinotPageSourceProvider.java

```diff
@@ -25,6 +25,7 @@
 import io.prestosql.spi.connector.ConnectorSplit;
```

Comment:
Maybe it instead of this pull request we should simply remove this deprecated method? How long ago was it deprecated?

Reply:
I agree, we can just remove it now.

Reply:
@iirekm How about it?